agent/src/main/java/com/microsoft/hydralab/agent/runner/monkey/AdbMonkeyRunner.java

```diff
@@ -58,18 +59,24 @@ protected void run(DeviceInfo deviceInfo, TestTask testTask, TestRun testRun) th
         /** run the test */
         reportLogger.info("Start monkey test");
         testRun.setTestStartTimeMillis(System.currentTimeMillis());
+        performanceService.testRunStarted();
         checkTestTaskCancel(testTask);
+        performanceService.testStarted("Adb monkey test");

```

Comment:
nit: use constant to replace magic number/string, etc.

Reply:
Fixed